packages/teams-js/src/internal/handlers.ts

```diff
@@ -94,14 +101,38 @@ export function callHandler(name: string, args?: unknown[]): [true, unknown] | [
   }
 }
 
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function registerHandlerWithVersion(
```

Comment:
Is it safe to assume that all of the "_____WithVersion" functions in this PR will replace the original implementations in a future iteration?

Reply:
No it will not replace the original implementation but "imagining all of APIs gonna use these. xxxxxWithVersion APIs, and then I will revert names back to the one without 'WithVersion'., which means those xxxxxWithVersion functions will no longer exist and original functions have no differences except there is an additional apiVersionTag parameter added to be passed in.